### PR TITLE
Add a second greeting header

### PR DESCRIPTION
**Pull request by andreamll**
Finish the file

### CODE REVIEW

**Review by Carlos-Sibaja — COMMENTED** [review]
Great Andrea!

**Review by Carlos-Sibaja — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -5,5 +5,6 @@
 </head>
 <body>
 <h1> Hey here Carlos from my laptop </hi>
+<h2> Hello, it's me, Andrea! </h2>
```
Comment (Carlos-Sibaja) [inline index.html]: Obrigado,  Andrea could you please add?  "I am from Brazil"

**Review by Carlos-Sibaja — CHANGES_REQUESTED** [review]
Ola,  Please change de color of your title to red.

File: index.html (hunk 1 above)
Comment (andreamll) [reply index.html]: Sure, I'll do that.

File: index.html [hunk 2]
```diff
@@ -5,6 +5,6 @@
 </head>
 <body>
 <h1> Hey here Carlos from my laptop </hi>
-<h2> Hello, it's me, Andrea! </h2>
+<h2 style="color: red;"> Hello, it's me, Andrea! I'm from Brazil</h2>
```
Comment (Carlos-Sibaja) [inline index.html:8]: Excelente!

**Review by Carlos-Sibaja — CHANGES_REQUESTED** [review]
Please proceed with the rest of the steps and add more content to the page.

**Pull request merged**